permissions/api_permissions/drf_permissions.py

```diff
@@ -254,33 +252,36 @@ def check_permissions_for_single(self, request, status, application_event_id):
         if not application_event_id:
             return False
         try:
-            service_sector = ServiceSector.objects.get(
-                applicationround=ApplicationRound.objects.get(
-                    applications=Application.objects.get(
+            service_sector = ServiceSector.objects.filter(
+                applicationround=ApplicationRound.objects.filter(
+                    applications=Application.objects.filter(
                         application_events=application_event_id
-                    )
-                )
+                    )[:1]
```

Comment:
What's the reason for slicing here? 
I see we're using `.first()` in lower section of the code for this QS, shouldn't that be fine without need to slice :thinking: 

Also seeing slicing used in other parts of the code too, so extra interested :nerd_face:

Reply:
Slicing is for the filter queries, that they will always return only 1 (even though we filter by IDs to foolproof the query).

This optimization to use `.filter()` instead of `.get()` is to keep these queries lazy as they will not necessarily be used therefore saving queries to the database.

Reply:
So, slicing is for the "subqueries" `.first()` is for the final call to database for a result.

Reply:
Yup, the filter is good way to go. The slicing might be unnecessary now that I think of it more but its fine